extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -498,6 +521,7 @@ public void handle() throws InterruptedException, ExecutionException, TimeoutExc
     public void handle() throws InterruptedException, ExecutionException, TimeoutException
     {
       consumer.close();
+      lagComputingConsumer.close();
```

Comment:
it can throw exception, the exception will be handled by the thread running the notice.